tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -646,11 +649,12 @@ def _get_param_inlays(
                 position=decl.rng.end,
                 label="param value is " + str(param),
                 padding_left=True,
+                kind=InlayHintKind.Parameter
```

Comment:
Dubious -- chapel 'param' is not the same as 'parameter' in conventional usage. 'param annotation' are specifically neither types nor parameters ("parameter annotation" I think is the `f(a=b)` inlay). I deliberately left the `kind` setting out of the inlay data.

Reply:
I will pull the InlayHintKind from this PR, it doesn't do much anyways (at least in vscode)